pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -2747,6 +2739,21 @@ public void checkInactiveSubscriptions() {
         }
     }
 
+    @VisibleForTesting
+    public void checkInactiveSubscriptionsWithExpirationTime(long expirationTimeMillis) {
```

Comment:
I don't think we should make it to be `PUBLIC`

Reply:
+1